apps/server/src/modules/loadDemo.ts

```diff
@@ -0,0 +1,24 @@
+import { copyFileSync } from 'fs';
+import { pathToStartDemo, resolveDemoDirectory, resolveDemoPath } from '../setup.js';
+import { ensureDirectory } from '../utils/fileManagement.js';
+
+/**
+ * @description ensures directories exist and populates stylesheet
+ * @return {string} - path to stylesheet file
+ */
+export const populateDemo = () => {
+  const demoInDisk = resolveDemoPath;
+  ensureDirectory(resolveDemoDirectory);
+
+  // eaven if demoInDisk exist we want to use startup demo
+  try {
+    demoInDisk.forEach((to, index) => {
+      const from = pathToStartDemo[index];
+      copyFileSync(from, to);
+    });
+  } catch (_) {
+    /* we do not handle this */
+  }
+
+  return demoInDisk;
```

Comment:
question: what does the return do? it seems mostly unused?

Reply:
yes I don't know 
I just kept the same format as the other loader, bu i can remove it from them both?

Reply:
I guess if they are not used we should remove it